src/singletons/Settings.hpp

```diff
@@ -125,6 +125,8 @@ class Settings : public ABSettings, public ConcurrentSettings
         "/behaviour/autocompletion/smallStreamerLimit", 1000};
     BoolSetting prefixOnlyEmoteCompletion = {
         "/behaviour/autocompletion/prefixOnlyCompletion", true};
+    BoolSetting userOnlyCompletionWithAt = {
+        "/behaviour/autocompletion/userOnlyCompletionWithAt", true};
```

Comment:
I believe this should be called `userCompletionOnlyWithAt` but otherwise seems good.

Reply:
Changed